x-pack/plugins/spaces/public/management/edit_space/manage_space_page.test.tsx

```diff
@@ -19,6 +19,13 @@ import { notificationServiceMock, scopedHistoryMock } from 'src/core/public/mock
 import { featuresPluginMock } from '../../../../features/public/mocks';
 import { Feature } from '../../../../features/public';
 
+// TODO: Temporary EUI mock
```

Comment:
Is there an issue open to track when it's safe to remove this mock? If so, can we link it here?